Tools/yocto/meta-openembedded_and_meta-webkit.patch

```diff
@@ -0,0 +1,42 @@
+
+diff --git a/sources/meta-openembedded/meta-python/recipes-devtools/python/python3-twisted_24.3.0.bb b/sources/meta-openembedded/meta-python/recipes-devtools/python/python3-twisted_24.3.0.bb
+index e5223cc..1454b62 100644
+--- a/sources/meta-openembedded/meta-python/recipes-devtools/python/python3-twisted_24.3.0.bb
++++ b/sources/meta-openembedded/meta-python/recipes-devtools/python/python3-twisted_24.3.0.bb
+@@ -15,7 +15,7 @@ do_install:append() {
+     find ${D} \( -name "*.bat" -o -name "*.c" -o -name "*.h" \) -exec rm -f {} \;
+ }
+ 
+-PACKAGES += "\
++PACKAGES =+ "\
+     ${PN}-zsh \
```

Comment:
Is this change needed?
The string appended already starts by an space (several actually)

Reply:
Yeah, I cherry-picked that patch from 'main' branch (not in scarthgap branch at present). Otherwise python3-twisted fails..